src/themes.js

```diff
@@ -40,15 +40,21 @@ export default {
      * @returns {Object}
      */
     getConfig: function getConfig() {
-        var initialConfig;
-
+        // set theme config from module.config() if it is not defined
         if (!themesConfig) {
-            initialConfig = module.config();
-            themesConfig = _.cloneDeep(initialConfig);
+            this.setConfig(module.config());
         }
         return themesConfig;
     },
 
+    /**
+     * Set themes config
+     * @param {Object} config Themes config
+     */
+    setConfig(config) {
```

Comment:
if this sets only the theme config then I'd call it setThemeConfig

Reply:
Based on our discussion, I will keep the name of the function.